src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CreateCohortWithDraftApprenticeshipController.cs

```diff
@@ -93,6 +95,7 @@ public async Task<IActionResult> AddDraftApprenticeship(AddDraftApprenticeshipVi
         [HttpGet]
         [Route("add-select-employer")]
         [Route("add/select-employer")]
+        [DasAuthorize(ProviderFeature.ProviderCreateCohortV2)]
```

Comment:
I remember seeing us add the Feature toggle in PAS. But did we add this FeatureToggle in ProviderCommitments?